config/jobs/kubeflow/kubeflow-periodics.yaml

```diff
@@ -14,3 +14,18 @@ periodics:
         value: kubeflow
       - name: BRANCH_NAME
         value: v0.3-branch
+- name: kubeflow-periodic-e2e
```

Comment:
Can we just call this "master" since it runs against the master branch? The prow config contains unit tests as well.